client/state/data-layer/wpcom/wordads/settings/index.js

```diff
@@ -71,6 +71,8 @@ export const saveWordadsSettings = ( action ) => ( dispatch, getState ) => {
 				enable_header_ad: settings.display_options.enable_header_ad,
 				wordads_second_belowpost: settings.display_options.second_belowpost,
 				wordads_inline_enabled: settings.display_options.inline_enabled,
+				wordads_bottom_sticky_enabled: settings.bottom_sticky_enabled,
+				wordads_sidebar_sticky_right_enabled: settings.sidebar_sticky_right_enabled,
```

Comment:
```suggestion
				wordads_bottom_sticky_enabled: settings.display_options.bottom_sticky_enabled,
				wordads_sidebar_sticky_right_enabled: settings.display_options.sidebar_sticky_right_enabled,
```

Reply:
> I think this is why you're not seeing the settings saved correctly. They are part of the display_options object.
Thank you! I stared at this and just couldn't see it 😅.